src/minidump.rs

```diff
@@ -767,6 +768,12 @@ impl MinidumpModuleList {
     }
 }
 
+impl Default for MinidumpModuleList {
```

Comment:
This seems fine. I'm not sure that there are really valid semantics for `Default` here, but considering that `new` already hands out an empty list this isn't much different.